### PR TITLE
libebml: update to 1.4.4

### PR DESCRIPTION
**Pull request by robxnano**

### CODE REVIEW

File: dev-libs/libebml/libebml-1.4.4.recipe [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+HOMEPAGE="https://github.com/Matroska-Org/libebml/"
+COPYRIGHT="2005-2023 Matroska"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="$HOMEPAGE/archive/release-$portVersion.tar.gz"
+CHECKSUM_SHA256="3b30d89a5699fef4bdf5b4d7de7a1c94d9be47fababf9caca59250282f051283"
+SOURCE_DIR="libebml-release-$portVersion"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
```
Comment (Begasus) [inline dev-libs/libebml/libebml-1.4.4.recipe:12]: No need for x86_gcc2 here:
```diff
-SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+SECONDARY_ARCHITECTURES="x86"
```

File: dev-libs/libebml/libebml-1.4.4.recipe [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+	cmake .. $cmakeDirArgs -DBUILD_SHARED_LIBS=TRUE \
+		-DCMAKE_BUILD_TYPE=Release
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make -C build install
+
+	prepareInstalledDevelLibs libebml
```
Comment (Begasus) [inline dev-libs/libebml/libebml-1.4.4.recipe:52]: Only 1 library involved, so:
```diff
-	prepareInstalledDevelLibs libebml
+	prepareInstalledDevelLib libebml
```

**Review by Begasus — CHANGES_REQUESTED** [review]

**Comment by Begasus** [comment]
Thanks so far, could you check if this is being used by others (aside of libmatroska)?
If not the preceeding recipe could be removed.

**Review by Begasus — APPROVED** [review]

**Comment by robxnano** [comment]
> Thanks so far, could you check if this is being used by others (aside of libmatroska)? If not the preceeding recipe could be removed.

Looks like it's used by vlc and gerbera. Should I test building them with the new version?

**Comment by Begasus** [comment]
SONAME didn't change, so should be good to remove the 1.4.2 recipe with this, Gerbera is still somewhat broken, VLC launches OK after an update here for libebml and libmatroska.

**Comment by Begasus** [comment]
Will do a quick check on 64bit with the older packages still installed for VLC, on 32bit in a bit, it crashes for me (maybe unrelated).

**Comment by Begasus** [comment]
Thanks!

**Pull request merged**